Pages/Sitemap.cshtml.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using QCUtilities.Interfaces;
+using QCVault.Utilities.Entities;
+using QCVault.Utilities.Services;
+
+namespace QCVault.Pages
+{
+    public class SitemapModel : PageModel
```

Comment:
Add `[ResponseCache(Duration = 3600)]` so it'll be picked up and cached by the proxy.